cypress/support/commands.js

```diff
@@ -197,6 +197,49 @@ Cypress.Commands.add('empiricaLoginPlayer', (playerKey) => {
       {log: false}
   )
 
+  log.snapshot("after");
+  log.end();
+})
+
+Cypress.Commands.add('empiricaLoginMultiPlayers', (playerKeys) => {
+  // if not already logged in, logs in
+  // TODO: someday, do this step programmatically
+
+  const log = Cypress.log({
+    name: "empiricaLoginMultiPlayers",
+    displayName: "🐘 Login Players",
+    message: playerKeys,
+    autoEnd: false,
+  });
+
+  cy.viewport(2000, 1000, { log: false })
+  let url = "/?"
+  cy.visit(`/?playerKey=${playerKeys[0]}&secondaryPlayerKey=${playerKeys[1]}&multiplayer=true`, { log: false });
+  cy.wait(300, { log: false })
+  log.snapshot("before");
+
+
+  //consent
+  cy.contains("consent", { timeout: 5000, log: false });
+  cy.contains("you may engage in video, audio, or text chat", { log: false });  // check IRB language present
+  cy.contains("We may share recordings under a confidentiality agreement", { log: false });  // check IRB language present
+  cy.contains("deliberation-study@wharton.upenn.edu", { log: false });  // check contact info present
+  cy.get("button", { log: false }).contains("I AGREE", { log: false }).click({ log: false });
```

Comment:
how do you distinguish between the two players? I wonder if we could set an id at the gameframe level, and then first get the right frame, and then take actions within the frame? Something like:
`cy.get([id='p1Frame']).get('button').etc()`